doc/nfs/NFS.md

```diff
@@ -0,0 +1,26 @@
+# Beholder Eye - NFS
+
+This beholder eye keep an eye into NFS, if it was mounted, if is possible read, write and delete files into it.
+
+## Exemple of use
+
+<code>
+require 'vendor/autoload.php';
+
+$conf = [
+  'eyes' => [
+      'Nfs' => [ // This Nfs is juts an alias, it can be anything
```

Comment:
This Nfs is **juts** an alias, it can be anything
This Nfs is **just** an alias, it can be anything